allauth/socialaccount/providers/atlassian/provider.py

```diff
@@ -0,0 +1,38 @@
+from allauth.socialaccount.providers.base import ProviderAccount
+from allauth.socialaccount.providers.oauth2.provider import OAuth2Provider
+
+from .views import AtlassianOAuth2Adapter
+
+
+class AtlassianAccount(ProviderAccount):
+    def get_profile_url(self):
+        return self.account.extra_data.get("picture")
+
+
+class AtlassianProvider(OAuth2Provider):
+    id = "atlassian"
+    name = "Atlassian"
+    account_class = AtlassianAccount
+    oauth2_adapter_class = AtlassianOAuth2Adapter
+
+    def extract_uid(self, data):
+        return data["account_id"]
+
+    def extract_common_fields(self, data):
+        return {
+            "email": data.get("email"),
+            "name": data.get("name"),
+            "username": data.get("nickname"),
+            "email_verified": data.get("email_verified"),
```

Comment:
**Question**: I don't see this field being documented, nor is it present in your test response. Does this property exist?

https://developer.atlassian.com/cloud/jira/platform/oauth-2-3lo-apps/#how-do-i-retrieve-the-public-profile-of-the-authenticated-user-

Reply:
Sorry, my bad. I didn't checked the example on the official documentation with the actual call.
The field exists, so added to the test response.